### PR TITLE
Update the dev channel for manual testing 

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: sample/development/components/MainScene.brs [hunk 1]
```diff
@@ -57,45 +66,71 @@ sub init()
...
+sub _sendEventWithCallback()
+  '----------------------------------------
+  ' Send an Experience Event with callback
+  '----------------------------------------
+
+  m.adobeEdgeSdk.sendEvent({
+    "eventType": "commerce.orderPlaced",
+    "commerce": {
+      "key3": "value3"
+    }
```
Comment (addb) [inline sample/development/components/MainScene.brs:80]: can you add identityMap as well?

```
{
      "eventType": "commerce.orderPlaced",
      "commerce": {
        "key3": "value3"
      }
      "identityMap": {
            "RIDA" : [
                  {
                        "id" : "SampleAdId",
                        "authenticatedState": "ambiguous",
                        "primary": false
                  }
              ]
      }
}
```

File: sample/development/components/MainScene.brs [hunk 2]
```diff
@@ -42,6 +50,7 @@ sub init()
   'configuration[ADB_CONSTANTS.CONFIGURATION.EDGE_DOMAIN] = ""
   m.adobeEdgeSdk.updateConfiguration(configuration)
 
+
```
Comment (addb) [inline sample/development/components/MainScene.brs:53]: nit: Let's remove the commented out code?

File: sample/development/components/MainScene.brs [hunk 3]
```diff
@@ -57,45 +66,71 @@ sub init()
...
 end sub
 
-sub onButtonSelected()
-  'SendEvent button pressed
-  if m.ButtonGroup.buttonSelected = 0
-    '----------------------------------------
-    ' Send an Experience Event with callback
-    '----------------------------------------
+sub _testShutdownAPI()
+  if m.adobeEdgeSdk = invalid
```
Comment (addb) [inline sample/development/components/MainScene.brs:105]: should we break this down to

setupSDK()
configure()
clearConfiguration()
SendEvent()
SendEventWithCallback()
shutDown()

this way we can clear the config and keep calling sendevent whcih will queue the events and call shutdown to test the scenario.

What do you think?

Comment (yangyansong-adbe) [reply sample/development/components/MainScene.brs:105]: I intend to keep the valid configuration and let the SDK task node (thread) keep sending edge requests. Then we can verify if the shutdown operation can deactivate the task node and drop requests correctly.

**Review by addb — APPROVED** [review]

**Pull request merged**